### ISSUE TITLE
Fails to store settings in Manjaro Linux

### ISSUE THREAD

**Issue by mominul** [OpenBangla/OpenBangla-Keyboard#17]
Topbar is always failing to store layout path setting.

**Comment by mominul** [comment]
Ah, the investigation hasn't been stopped folks!
If I am not getting things wrong, the main scenario happening is this:
In KDE environment, Qt automatically is inserting `&` in Layout names in the layout menu name so user can select the items pressing <kbd>alt</kbd> key.
If Layout name is `&Bornona`, user can select the method by pressing <kbd>alt</kbd> + <kbd>b</kbd> because the `&` symbol is present before `B`.
Long story short: the insertion of `&` character is messing up the whole thing. :boom:

As I have committed the work workaround, there shouldn't any problems for now but I will try to get myself an another solution.

**Comment by mominul** [comment]
I think I have figured out the main thing:
From Qt version 5.10, Qt automatically adds shortcuts to menu items. For that Qt includes a `&` character. So when we use `QAction::text()` function, we get a string including a `&` character and we mess all things up.

So this is not a KDE environment based issue. Most distributions use Qt 5.9 like Ubuntu 17.10 and so we didn't hit this issue. But Arch distributions use latest Qt and we hit this issue really hard :boom:
I am working on a new commit which will simplify that workaround.
